packages/containers/src/List/List.container.js

```diff
@@ -97,7 +98,7 @@ class List extends React.Component {
 					api.action.getActionCreatorFunction(
 						this.context,
 						this.props.actions.title,
-					)(p.id),
+					)(e, p, this.context),
```

Comment:
@acateland the first context is to retrieve the function that will be called.
context.registry.
the second is to call this function which doesnt' know the context.
We have a convention in CMF which said every action provider should accept
event, data, context
as arguments.

To remove this it would means modify the action creator by creating a wrapper around it to inject the context. I don't like this because you don't know the API of the action creator (this is just a convention)